tests/unit/core/components/TestAMInstallButton.js

```diff
@@ -640,4 +646,45 @@ describe(__filename, () => {
 
     expect(root.find(Button)).toHaveProp('disabled', false);
   });
+
+  describe('install_button_color experiment', () => {
+    const renderWithExperiment = (props = {}) => {
+      return render({
+        experimentEnabled: true,
```

Comment:
I guess I hadn't noticed this pattern before but I think it would be nicer to test it with a config object. That would cover the integration between our config parameter and the experiment (which is important).

```js
return render({
  _config: getFakeConfig({ experiments: { install_button_color: true } }),
  ...props,
});
```